internal/backends/qt/build.rs

```diff
@@ -29,19 +30,11 @@ fn main() {
     }
 
     let mut config = cpp_build::Config::new();
-
+    for f in std::env::var("DEP_QT_COMPILE_FLAGS").unwrap().split_terminator(";") {
+        config.flag(f);
+    }
     config.flag_if_supported("-std=c++17");
     config.flag_if_supported("/std:c++17");
```

Comment:
We still need to enable C++17 manually because qttypes only use this flag for Qt >= 6 , but not for Qt5